src/ast/variables/SyntheticNamedExport.ts

```diff
@@ -0,0 +1,25 @@
+import Module, { AstContext } from '../../Module';
```

Comment:
One minor thing: Maybe call this a ...Variable so that it fits the names of the other variable types

Reply:
done!